src/ZfcUser/Factory/Form/ChangeEmail.php

```diff
@@ -0,0 +1,44 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * User: Clayton Daley
+ * Date: 3/10/2015
+ * Time: 9:34 AM
+ */
+
+namespace ZfcUser\Factory\Form;
+
+use Zend\Form\FormElementManager;
+use Zend\ServiceManager\FactoryInterface;
+use Zend\ServiceManager\ServiceLocatorInterface;
+use ZfcUser\Form;
+use ZfcUser\Validator;
+
+class ChangeEmail implements FactoryInterface
+{
+    public function createService(ServiceLocatorInterface $formElementManager)
+    {
+        if ($formElementManager instanceof FormElementManager) {
```

Comment:
Legacy code doesn't necessarily construct the class from the FEM.  We need to check if the ServiceLocator is actually the FEM or just the regular ServiceManager.  In the latter case, we need to take the initiative to obtain the FEM.